### ISSUE TITLE
Enum support

### ISSUE THREAD

**Issue by arnaudauroux** [geeklearningio/gl-swagger-generator#9]
Enumeration are today converted to string

**Comment by sandorfr** [comment]
Yes and no it depends on your swagger / api. For instance it's likely you are using this swashbuckle settings: https://github.com/domaindrivendev/Swashbuckle.AspNetCore#customize-schema-for-enum-types.

That being said, I definitely want to improve the scenarios around enums which shouldn't be to hard especially since typescript now supports string enums. 

Can you tell me more about what you expect so I might come with a draft spec?
  
other inputs in #40

**Comment by sandorfr** [comment]
ping @arnaudauroux
